components/mailgun/mailgun.app.js

```diff
@@ -0,0 +1,58 @@
+const axios = require("axios");
+const formData = require("form-data");
+const Mailgun = require("mailgun.js");
+
+module.exports = {
+  type: "app",
+  app: "mailgun",
+  methods: {
+    api (api) {
+      const mailgun = new Mailgun(formData);
+      const mg = mailgun.client({
+        username: "api",
+        key: this.$auth.api_key,
+        public_key: this.$auth.api_key,
+      });
+      return mg[api];
+    },
+    async suppress (domain, type, suppression) {
```

Comment:
Is `suppress` not available via the JS client lib?

Reply:
@dylburger heh, it is but no matter what I tried I couldn't get it to work. I think it must have a bug, or Mailgun changed their API behavior and their official client library hasn't caught up. Whatever it is, I noticed that it is particularly touchy about the Content-Type header and the exact format of the request body.